packages/unchained-client/src/bitcoin/parser/index.ts

```diff
@@ -15,11 +16,12 @@ export class TransactionParser {
 
   constructor(args: TransactionParserArgs) {
     this.chainId = args.chainId
+    const assetReference = args.assetReference || ASSET_REFERENCE.Bitcoin
```

Comment:
Ok for now, would like to make a utxo parser and follow similar pattern to evm parser long term (parser knows what chain and can set the `assetId`).

Also stylistic pref for `??` over `||` on undefined default fallback check